From 33152efd07626d87a5cc672d26ae5825869318bb Mon Sep 17 00:00:00 2001
From: Yufei Gu <yufei.gu@cloudera.com>
Date: Thu, 12 Jan 2017 15:00:40 -0800
Subject: [PATCH 2146/2848] MAPREDUCE-5335. Rename Job Tracker terminology in
 ShuffleSchedulerImpl. Contributed by Devaraj K.

(cherry picked from commit b42d09eb62bd1725d70da59f1a6fdac83cea82d1)
(cherry picked from commit 9129b3415c265c59cd25d0ec976e5a838ecc7089)

Change-Id: I3d02e92b8e127f070ace2a452f157e3b26bc446f
---
 .../task/reduce/ShuffleSchedulerImpl.java          |    9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/task/reduce/ShuffleSchedulerImpl.java b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/task/reduce/ShuffleSchedulerImpl.java
index abdff34..7489e63 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/task/reduce/ShuffleSchedulerImpl.java
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/task/reduce/ShuffleSchedulerImpl.java
@@ -301,7 +301,8 @@ public synchronized void copyFailed(TaskAttemptID mapId, MapHost host,
       }
     }
 
-    checkAndInformJobTracker(failures, mapId, readError, connectExcpt, hostFail);
+    checkAndInformMRAppMaster(failures, mapId, readError, connectExcpt,
+        hostFail);
 
     checkReducerHealth();
 
@@ -333,15 +334,15 @@ public void reportLocalError(IOException ioe) {
     reporter.reportException(ioe);
   }
 
-  // Notify the JobTracker
+  // Notify the MRAppMaster
   // after every read error, if 'reportReadErrorImmediately' is true or
   // after every 'maxFetchFailuresBeforeReporting' failures
-  private void checkAndInformJobTracker(
+  private void checkAndInformMRAppMaster(
       int failures, TaskAttemptID mapId, boolean readError,
       boolean connectExcpt, boolean hostFailed) {
     if (connectExcpt || (reportReadErrorImmediately && readError)
         || ((failures % maxFetchFailuresBeforeReporting) == 0) || hostFailed) {
-      LOG.info("Reporting fetch failure for " + mapId + " to jobtracker.");
+      LOG.info("Reporting fetch failure for " + mapId + " to MRAppMaster.");
       status.addFetchFailedMap((org.apache.hadoop.mapred.TaskAttemptID) mapId);
     }
   }
-- 
1.7.9.5

